src/chapters/arrays/array-methods.rst

```diff
@@ -0,0 +1,151 @@
+.. _array-methods:
+
+Array Methods
+==============
+
+.. index::
+   single: array; methods
+
+As with strings, JavaScript provides us with useful **methods** for arrays.
+Unlike strings, arrays are *mutable* and can be changed.
+
+Some methods alter an array, while others *return* values. For example,
+``console.log([1, 2, 3].includes(2))`` prints ``true``. The  action
+``[1, 2, 3].includes(2)`` checks the array to see if it contains the item ``2``
+and then *returns* a boolean which can be printed (or assigned to a variable).
+
+Common Array Methods
+--------------------
+
+Here is a sample of the most frequently used array methods. More complete lists
+can be found here:
+
+#. Novice list: `W3 Schools Array Methods <https://www.w3schools.com/jsref/jsref_obj_array.asp>`__
+#. More detailed list: `MDN Web Docs <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array>`__
+
+To see detailed examples for a particular method, control-click
+(or right-click) on its name.
+
+.. list-table:: Methods That Return Information About The Array
+   :header-rows: 1
+
+   * - Method
+     - Syntax
+     - Description
+   * - :ref:`includes <includes-examples>`
+     - ``arrayName.includes(item)``
+     - Checks if an array contains the specified item.
+
+   * - :ref:`indexOf <indexOf-examples>`
+     - ``arrayName.indexOf(item)``
+     - Returns the index of the FIRST occurence of an item in the array. If the item is not in the array, -1 is returned.
+
+.. list-table:: Methods That Rearrange The Entries In The Array
+   :header-rows: 1
+
+   * - Method
+     - Syntax
+     - Description
+   * - :ref:`reverse <reverse-example>`
+     - ``arrayName.reverse()``
+     - Pretty straightforward. Reverses the order of the elements in an array.
```

Comment:
"Pretty straightforward" is unnecessary and may make a student who doesn't get it feel bad.

Reply:
I was wondering about that.  Fixed.